### ISSUE TITLE
Event admin api does not check for the organiser email address

### ISSUE THREAD

**Issue by se-bastiaan** [svthalia/concrexit#2578]
### Describe the bug
Since #2547 the check for the organiser email is no longer located in the model clean, therefore the model clean serializer does no longer check these values.

### How to reproduce
Steps to reproduce the behaviour:
1. Use the API
2. No clean

### Expected behaviour
Data should be cleaned.

**Comment by se-bastiaan** [comment]
Since I have no idea what the issue was that was solved in the mentioned PR, I'd just say: move this check back to the model clean. The events admin forms is not the right place for it.

**Comment by se-bastiaan** [comment]
Yes I just found out. Nobody opened an issue so I took the liberty :p

Maybe it would be good to generalise the issue for all admin stuff? 

For this specific case a completely different approach where we do not care about the email address being present could also be an option.

**Comment by ColonelPhantom** [comment]
The PR was called V2 because there already was a PR hoping to fix an issue like this: #2546.

Generalising sounds like a good idea, but I don't really know how :sweat_smile: doing it in the model is not possible, but we do want it to be sufficiently general.

I think not caring is a good workaround in this case, also for e.g. the case where one organiser has a list, but another one does not?
